 Network rinha-nginx-2024q1  Creating
 Network rinha-nginx-2024q1  Created
 Container marcoagpegoraro-db-1  Creating
 Container marcoagpegoraro-db-1  Created
 Container marcoagpegoraro-api02-1  Creating
 Container marcoagpegoraro-api01-1  Creating
 Container marcoagpegoraro-api02-1  Created
 Container marcoagpegoraro-api01-1  Created
 Container marcoagpegoraro-nginx-1  Creating
 Container marcoagpegoraro-nginx-1  Created
Attaching to api01-1, api02-1, db-1, nginx-1
db-1     | The files belonging to this database system will be owned by user "postgres".
db-1     | This user must also own the server process.
db-1     | 
db-1     | The database cluster will be initialized with locale "en_US.utf8".
db-1     | The default database encoding has accordingly been set to "UTF8".
db-1     | The default text search configuration will be set to "english".
db-1     | 
db-1     | Data page checksums are disabled.
db-1     | 
db-1     | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db-1     | creating subdirectories ... ok
db-1     | selecting dynamic shared memory implementation ... posix
db-1     | selecting default max_connections ... 100
db-1     | selecting default shared_buffers ... 128MB
db-1     | selecting default time zone ... UTC
db-1     | creating configuration files ... ok
db-1     | running bootstrap script ... ok
db-1     | sh: locale: not found
db-1     | 2024-02-16 12:06:28.769 UTC [30] WARNING:  no usable system locales were found
db-1     | performing post-bootstrap initialization ... ok
api01-1  | V panic: Connection to a PG database failed: could not connect to server: Connection refused
api01-1  | 	Is the server running on host "192.168.0.2" and accepting
api01-1  | 	TCP/IP connections on port 5432?
api01-1  | 
api01-1  | v hash: 9f532f1
api01-1  | backtrace_symbols is missing => printing backtraces is not available.
api01-1  | Some libc implementations like musl simply do not provide it.
api02-1  | V panic: Connection to a PG database failed: could not connect to server: Connection refused
api02-1  | 	Is the server running on host "192.168.0.2" and accepting
api02-1  | 	TCP/IP connections on port 5432?
api02-1  | 
api02-1  | v hash: 9f532f1
api02-1  | backtrace_symbols is missing => printing backtraces is not available.
api02-1  | Some libc implementations like musl simply do not provide it.
db-1     | syncing data to disk ... ok
db-1     | 
db-1     | 
db-1     | Success. You can now start the database server using:
db-1     | 
db-1     |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db-1     | 
db-1     | initdb: warning: enabling "trust" authentication for local connections
db-1     | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db-1     | waiting for server to start....2024-02-16 12:06:30.479 UTC [36] LOG:  starting PostgreSQL 16.2 on x86_64-pc-linux-musl, compiled by gcc (Alpine 13.2.1_git20231014) 13.2.1 20231014, 64-bit
db-1     | 2024-02-16 12:06:30.482 UTC [36] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-16 12:06:30.492 UTC [39] LOG:  database system was shut down at 2024-02-16 12:06:29 UTC
db-1     | 2024-02-16 12:06:30.498 UTC [36] LOG:  database system is ready to accept connections
db-1     |  done
db-1     | server started
db-1     | CREATE DATABASE
db-1     | 
db-1     | 
db-1     | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/script.sql
db-1     | CREATE TABLE
db-1     | CREATE TABLE
nginx-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
db-1     | CREATE INDEX
db-1     | CREATE INDEX
db-1     | psql:/docker-entrypoint-initdb.d/script.sql:20: NOTICE:  relation "idx_transacoes_id_desc" already exists, skipping
db-1     | CREATE FUNCTION
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
db-1     | INSERT 0 5
db-1     | 
db-1     | 
db-1     | 2024-02-16 12:06:31.789 UTC [36] LOG:  received fast shutdown request
db-1     | waiting for server to shut down....2024-02-16 12:06:31.836 UTC [36] LOG:  aborting any active transactions
db-1     | 2024-02-16 12:06:31.839 UTC [36] LOG:  background worker "logical replication launcher" (PID 42) exited with exit code 1
db-1     | 2024-02-16 12:06:31.839 UTC [37] LOG:  shutting down
db-1     | 2024-02-16 12:06:31.843 UTC [37] LOG:  checkpoint starting: shutdown immediate
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
db-1     | 2024-02-16 12:06:32.650 UTC [37] LOG:  checkpoint complete: wrote 942 buffers (5.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.487 s, sync=0.268 s, total=0.812 s; sync files=308, longest=0.173 s, average=0.001 s; distance=4307 kB, estimate=4307 kB; lsn=0/19250F8, redo lsn=0/19250F8
db-1     | 2024-02-16 12:06:32.657 UTC [36] LOG:  database system is shut down
db-1     |  done
db-1     | server stopped
db-1     | 
db-1     | PostgreSQL init process complete; ready for start up.
db-1     | 
db-1     | 2024-02-16 12:06:32.717 UTC [1] LOG:  starting PostgreSQL 16.2 on x86_64-pc-linux-musl, compiled by gcc (Alpine 13.2.1_git20231014) 13.2.1 20231014, 64-bit
db-1     | 2024-02-16 12:06:32.717 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db-1     | 2024-02-16 12:06:32.717 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db-1     | 2024-02-16 12:06:32.724 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-16 12:06:32.734 UTC [54] LOG:  database system was shut down at 2024-02-16 12:06:32 UTC
db-1     | 2024-02-16 12:06:32.741 UTC [1] LOG:  database system is ready to accept connections
nginx-1  | 2024/02/16 12:06:32 [error] 31#31: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.0.3:8080/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 12:06:32 [error] 31#31: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.0.4:8080/clientes/1/extrato", host: "localhost:9999"
api01-1 exited with code 0
api02-1 exited with code 0
nginx-1  | 2024/02/16 12:06:34 [error] 31#31: *4 no live upstreams while connecting to upstream, client: 192.168.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 12:06:36 [error] 31#31: *5 no live upstreams while connecting to upstream, client: 192.168.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 12:06:38 [error] 31#31: *6 no live upstreams while connecting to upstream, client: 192.168.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 12:06:40 [error] 31#31: *7 no live upstreams while connecting to upstream, client: 192.168.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 12:06:42 [error] 31#31: *8 no live upstreams while connecting to upstream, client: 192.168.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1 exited with code 0
nginx-1 exited with code 0
api01-1 exited with code 0
api01-1 exited with code 137
api02-1 exited with code 0
api02-1 exited with code 137
db-1     | 2024-02-16 12:11:09.818 UTC [1] LOG:  received fast shutdown request
db-1     | 2024-02-16 12:11:09.828 UTC [1] LOG:  aborting any active transactions
db-1     | 2024-02-16 12:11:09.831 UTC [1] LOG:  background worker "logical replication launcher" (PID 57) exited with exit code 1
db-1     | 2024-02-16 12:11:09.831 UTC [52] LOG:  shutting down
db-1     | 2024-02-16 12:11:09.835 UTC [52] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-02-16 12:11:09.982 UTC [52] LOG:  checkpoint complete: wrote 741 buffers (4.5%); 0 WAL file(s) added, 0 removed, 1 recycled; write=0.013 s, sync=0.115 s, total=0.152 s; sync files=30, longest=0.073 s, average=0.004 s; distance=19685 kB, estimate=19685 kB; lsn=0/2C5E720, redo lsn=0/2C5E720
db-1     | 2024-02-16 12:11:09.986 UTC [1] LOG:  database system is shut down
db-1 exited with code 0
